src/pages/AboutUs/Details.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import pic from './pic.png';
+import { MDBContainer, MDBRow, MDBCol } from 'mdbreact';
+
+function Details() {
+  return (
+    <MDBContainer className="bg-gray-100">
+      <MDBRow>
+        <MDBCol lg="5">
+          <img src={pic} alt="artwork of a girl" />
+        </MDBCol>
+        <MDBCol>
+          <div className="mt-40">
+            <h3 className="font-weight-bold  p-0  text-3xl ml-1">
```

Comment:
why did you have two spaces here ?
can you make it like this? 
```suggestion
            <h3 className="font-weight-bold p-0 text-3xl ml-1">
```